src/main/java/com/acrolinx/sidebar/pojo/settings/PluginSupportedParameters.java

```diff
@@ -26,6 +28,13 @@ public PluginSupportedParameters(boolean checkSelection, boolean supportsBatchCh
         this.supportsBatchChecks = supportsBatchChecks;
     }
 
+    public PluginSupportedParameters(boolean checkSelection,boolean supportsBatchChecks,boolean supportsLive)
+    {
+        this.checkSelection = checkSelection;
+        this.supportsBatchChecks = supportsBatchChecks;
+        this.supportsLive = supportsLive;
+    }
+
```

Comment:
The constructors can also be chained so that each constructor initializes all three fields. The default value for a boolean is false anyway but better to initialize explicitly.

Reply:
chained the constructors👍🏻